<?xml version="1.0"?>
<launch>
  <node pkg="undock_drive" type="undock_drive_action_server.py" name="undock_drive_action">
    <!--  in the action message you need to define a target position.
          To get the current position of the robot a tf lookup between
          global_frame and base_frame is used. -->
    <param name="global_frame" value="map"/>
    <param name="base_frame" value="base_link"/>
    
    <!-- this is the topic name where the Twist messages are send to -->
    <param name="velocity_cmd_topic" value="mobile_base/commands/velocity"/>
    
    <!-- the distance in meter a goal is considered as reached -->
    <param name="target_reached_distance" value="0.2"/>
    
    <!-- angular difference in rad the goal angle is considered as reached -->
    <param name="target_reached_angle" value="0.26"/>
  </node>
</launch>
